### ISSUE TITLE
D8024D53 - Gears of War

### ISSUE THREAD

**Issue by deadair3210** [xenia-project/game-compatibility#169]
Version Tested - https://github.com/benvanik/xenia/pull/417

Marketplace - http://marketplace.xbox.com/en-US/Product/Gears-of-War/66acd000-77fe-1000-9115-d8024d5307d5

Log - https://gist.github.com/deadair3210/ad4d6810033a6bd71a6c

Game launches, plays the Epic Games logo, plays the sound for the Microsoft Logo, Shows the title screen and plays the first segment of music before freezing

![gearsofwar](https://cloud.githubusercontent.com/assets/1053504/9720449/cc7527b6-5553-11e5-8152-728bcb362a41.PNG)

**Comment by Razzile** [comment]
state hang sound right?

**Comment by deadair3210** [comment]
Sounds Right

**Comment by MWorksGH** [comment]
Anyone interested in making this game work? We can create a team to achieve that!.

**Comment by Sakxz13** [comment]
Someone can teach me coding in the code that uses Xenia?

**Comment by win98and95** [comment]
Tested on (https://github.com/xenia-project/xenia/commit/952819ed87c29a81f55db1eca87bb20e187c95e7)
With D3D12 still hangs at the title screen
[xenia.zip](https://github.com/xenia-project/game-compatibility/files/2676670/xenia.zip)

**Comment by robzombie91** [comment]
Tested on (xenia-project/xenia@61fd5a6d)
With D3D12 and VULKAN, game still hangs on title screen

**Comment by Prism019** [comment]
The TitleID is incorrect. it should be 4D5307D5.

**Comment by Emogop** [comment]
![Screenshot_618](https://user-images.githubusercontent.com/42875723/87474927-89253880-c62c-11ea-9a15-0ec3bb4c8dd0.png)
It's Ingame now, graphics is broken.

**Comment by Kazmakistan** [comment]
Tested with Vulkan, title screen black and white. No red animation in background. 

Tested with Direct X, game was somewhat playable. Voices sounded great but lots of missing textures, walls, characters. Not sure if I just need to wait for shaders to be created.

**Comment by slick949** [comment]
I am new to github but I just wanted to add to the thread that there is zero sound during any cutscenes. Game only works on canary with patches and still no sound

**Comment by bomabomaboma** [comment]
> I am new to github but I just wanted to add to the thread that there is zero sound during any cutscenes. Game only works on canary with patches and still no sound

Can you attach a log and drop your specs?
I did not have the same problem.

Game was playable for me, albeit with broken ambient occlusion and other, smaller graphical defects.

**Comment by slick949** [comment]
> Can you attach a log and drop your specs?

idk how to create a log 
specs are:
Ryzen 7 1700 
16gb ram
XFX RX 580

**Comment by metalman1988** [comment]
Sound is really messed up in some cut scenes and gameplay.  Here's a log
[xenia.log](https://github.com/xenia-project/game-compatibility/files/6581721/xenia.log)

**Comment by backgamon** [comment]
Using PAL (with Fr support), it seem to work enough here.
Some desync and mute audio in some cutscene(only with game engine, FMV are fine).
Some blacks blocks are visibles in the same cutscene, mainly on characters.
Used Game for about 2h without any issue during gameplay.
Using RTv/DSV2x.
![2](https://user-images.githubusercontent.com/39404647/120534925-ae589c00-c3e2-11eb-80b4-dcddafdf1405.jpg)
![3](https://user-images.githubusercontent.com/39404647/120534948-b44e7d00-c3e2-11eb-88c3-73bc2fd71639.jpg)
![4](https://user-images.githubusercontent.com/39404647/120534960-b6b0d700-c3e2-11eb-8058-940efd24c137.jpg)


I9 10850k@4.8/4.9Ghz
Asus Maximus XII Formula
64Go DDR4 GSkill Trident Z 3600Mhz[XMPII]
Gigabyte 2070 RTX 8G OC
Windows 10 Pro x64

**Comment by git-tiger** [comment]
Strange audio bug during cutscenes. When games starts, and inside the jail, there is a loud pitch background noise.
Come again in other cutscenes. Okay during gameplay.

**Comment by HexMoon87** [comment]
![image](https://user-images.githubusercontent.com/106915127/172051302-7f31eab5-fcd3-46c0-92a5-f949a7f60b11.png)

Doors turning black

**Comment by lsmith7887** [comment]
I could be wrong, but the sound issue seems to occur with the same audio cues every time, regardless of performance. Not sure if this info will help figure out a fix.

**Comment by Malice27** [comment]
Sound issue is fixed in the latest xenia canary 1.0.900

**Comment by LethalManBoob** [comment]
![Screenshot from 2023-04-02 19-20-24](https://user-images.githubusercontent.com/30273211/229371651-866a524b-cbf2-4749-9710-97095dcb5981.png)
on both master and canary. I'm using wine with vkd3d halp

**Comment by LethalManBoob** [comment]
heeeelp

**Comment by cleggorz** [comment]
Me and my mate been playing coop on dxd12 canary and having a blast! A few graphic errors and slowdowns but the game is running great.

**Comment by langestefan** [comment]
Has anyone tried getting this to run on steam deck? It starts up, I see the main menu and then after 3 seconds I get 'assertion failed', and 'nested exception' and it crashes.

**Comment by IceStar3030** [comment]
> Has anyone tried getting this to run on steam deck? It starts up, I see the main menu and then after 3 seconds I get 'assertion failed', and 'nested exception' and it crashes.

Yes, i just installed it on Deck. For starters it runs at 30fps, a little stuttery but could be shader saving at the beginning, however opacity is off with certain assets including the main character; you can see through your polygons, could be a texture or lighting issue but I will do some tweaks.

**Comment by SloaJay** [comment]
> Me and my mate been playing coop on dxd12 canary and having a blast! A few graphic errors and slowdowns but the game is running great.

Yep, just completed it on newest canary build. With ROV setting and shadow fix almost no glitches. Except couple. Works fine.

PS. In one scene where you gotta push car forward to gun nest at night. On first try car goes all the way to the end. If one get killd and start from checkpoint car moves a little bit less forward. And on next try it moves noticeably shorter distance and so on. So you gotta exit to menu and then load back to game to try again.

**Comment by jeffbezos890** [comment]
3070ti + i512600k
Few missions in, latest canary build with patches. With vsync off, 1440p upscale, heavy post processing --- stable 110ish fps. Stock bilinear filtering fluctuated from 150-270fps.
Absolutely insane performance.
Variable refresh rate on non capable monitors introduces screen tearing fyi big fix.
Occasional graphical issues like flickering blood pools. Otherwise this is sick frfr.

**Comment by raven1986** [comment]
I'm having issues at the beginning of the game audio freezing on sound effects when door to prison cell is opened up and wont stop.

**Comment by jeffbezos890** [comment]
> I'm having issues at the beginning of the game audio freezing on sound effects when door to prison cell is opened up and wont stop.

Make sure you're using the latest canary fork with patches. Xenia master did the same for me.

**Comment by Margen67** [comment]
Xenia Canary issue: https://github.com/xenia-canary/game-compatibility/issues/44
